### ISSUE TITLE
CDN help

### ISSUE THREAD

**Issue by jhaydtw** [wet-boew/wet-boew#8507]
I wish I knew more about this, but here is the issue from my developers:
We need the directory structure to map to the .css and .js files.
--------------------------
What do we include to get all these files? Including the datatables css and js files?

For example this is base directory we have:
https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_27/

Then we add  /cdts/compiled/soyutils.js  to get this file:
https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_27/cdts/compiled/soyutils.js
And this cdts/compiled/wet-en.js to get this one:
https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_27/cdts/compiled/wet-en.js

I accidentally figured out this jquery file I needed:  js/jquery/2.1.4/jquery.min.js
https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_27/js/jquery/2.1.4/jquery.min.js

But we need lots of other files, like for example : https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js
-----------------------------------------------------------------------

Is there a list somewhere?  Is there a place to view the cdn tree?

Thanks,
Jeff

**Comment by thekodester** [comment]
See https://datatables.net for more features of the plugin.

FYI: WET excludes some features for accessibility reason.

**Comment by thekodester** [comment]
See [table.js](https://github.com/wet-boew/wet-boew/blob/6dda28ad4d64de6a051cdd7811024a615066f414/src/plugins/tables/tables.js#L81) on how WET loads the table.

**Comment by RobJohnston** [comment]
The download would show you the relative paths: http://wet-boew.github.io/wet-boew/docs/versions/dwnld-en.html

But if I understand, you shouldn't be digging for individual JavaScript  and CSS files!  Look at the code-behind for any page and you'll see the links for the wet-boew.js files, for example, https://www.canada.ca/etc/designs/canada/wet-boew/js/wet-boew.min.js.

**Comment by thekodester** [comment]
Also if your developers need help there is a weekly code sprint usually on Tuesday. Imagine @duboisp could give you more information.

**Comment by duboisp** [comment]
@shawnthompson

**Comment by shawnthompson** [comment]
I don't think this is a CDTS issue. I think they are asking for all the WET JS files. @jhaydtw  Can you please tell us more about your issue?
